ush/python/pygw/src/tests/test_schema.py

```diff
@@ -0,0 +1,88 @@
+"""
+Description
+-----------
+
+Unit-tests for `pygw.schema`.
+
+"""
+
+# ----
+
+import os
+
+from pygw import schema
+from pygw.yaml_file import parse_yaml
+from pygw.schema import SchemaError
+from pygw.configuration import cast_strdict_as_dtypedict
+
+# ----
+
+# Define the path to the YAML-formatted file containing the schema
+# attributes.
+yaml_path = os.path.join(os.getcwd(), "tests",
+                         "test-files", "test_schema.yaml")
+data = parse_yaml(path=yaml_path)
```

Comment:
This will fail in CI.  Disable.
```suggestion
#yaml_path = os.path.join(os.getcwd(), "tests",
                         "test-files", "test_schema.yaml")
#data = parse_yaml(path=yaml_path)
```